examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -54,6 +54,18 @@ spec:
   selector:
     app: cockroachdb
 ---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: cockroachdb-budget
+  labels:
+    app: cockroachdb
+spec:
+  selector:
+    matchLabels:
+      app: cockroachdb
```

Comment:
Does  PodDisruptionBudget work for Deployments also ? Looking at http://kubernetes.io/docs/admin/disruptions/ its not obvious to me.